modules/ROOT/pages/installation/persistence.adoc

```diff
@@ -4,26 +4,32 @@ Server installation is a two-step process.
 The first step, as described in this section, is to set up a Neo4j DBMS to act as persistence storage for NOM.
 
 
-The NOM persistence DBMS is used to store all NOM data (topology, metrics, etc.).
+The NOM persistence DBMS is used to store all NOM data (topology, metrics, etc.)
 The DBMS and the NOM Server need to be running simultaneously, or the NOM Server will not work.
 
 
-== Self managed DBMS
+== Self-managed DBMS
 
 NOM comes with a license to run a Neo4j DBMS with up to 4 cores of CPU and up to 16 GB of RAM for the use as NOMs persistence.
 
 === Specifications
-* Neo4j Enterprise Edition 4.4 or 5.x
+* Neo4j Enterprise Edition 4.4, 5.x or 2025.x
 * At least 10 GB free disk space
 * For minimum requirements, see also link:/docs/operations-manual/current/installation/requirements/[Operations Manual -> System requirements].
 
 === Install and prepare
 
-* Please refer to the link:/docs/operations-manual/current/installation[Operations Manual -> Installation] for information on how to install Neo4j.
+Please refer to the link:/docs/operations-manual/current/installation[Operations Manual -> Installation] for information on how to install Neo4j.
 
-* Make sure to set a secure password for the built-in admin (`neo4j`) and/or create a specific user for the NOM Server.
-Be aware that that user needs full admin access.
-The login to the NOM Server is needed for server installation.
+Ensure that the Neo4j database to be used for NOM persistence fulfills the following requirements:
+
+* it is empty and does not contain any indexes or constraints,
+* Cypher 5 is selected as default language for this database. For example, this can be done by executing the statement `ALTER DATABASE neo4j SET DEFAULT LANGUAGE CYPHER 5`. See more details in link:/docs/cypher-manual/current/queries/select-version/#select-default-cypher-version[Cypher Manual -> Select Cypher version].
```

Comment:
```suggestion
* Cypher 5 is selected as default language for this database. 
For example, this can be done by executing the statement `ALTER DATABASE neo4j SET DEFAULT LANGUAGE CYPHER 5`. 
For more details see link:{neo4j-docs-base-uri}/cypher-manual/current/queries/select-version/#select-default-cypher-version[Cypher Manual -> Select Cypher version].
```